Widgets/widget-ActiveIncidentsByRole.json

```diff
@@ -1,11 +1,11 @@
 {
-  "id": "mean-time-to-resolution",
+  "id": "incidents-by-role",
```

Comment:
Id change. Please check.

Reply:
i deleted the `Mttr` widget, it is not a change of id